spec/lib/miq_environment_spec.rb

```diff
@@ -15,7 +15,7 @@
       end
 
       example "local_ip_address" do
-        expect(described_class.local_ip_address).to eq(`hostname -i`.chomp.split.first)
+        expect(Socket.ip_address_list.map(&:ip_address)).to include(described_class.local_ip_address)
```

Comment:
This test feels backwards...we should have the `described_class.local_ip_address` on the left...

```ruby
expect(described_class.local_ip_address).to be_in(Socket.ip_address_list.map(&:ip_address))
```

Reply:
:+1: nice, fixed